packages/expo-screen-capture/ios/ScreenCaptureModule.swift

```diff
@@ -69,4 +98,48 @@ public final class ScreenCaptureModule: Module {
       "body": nil
     ])
   }
+
+  private func preventScreenshots() {
+    guard #available(iOS 13.0, *) else {
+      return
+    }
+
+    guard let keyWindow = UIApplication.shared.keyWindow,
+          let visibleView = keyWindow.subviews.first else { return }
+
+    let textField = UITextField()
+    textField.isSecureTextEntry = true
+    textField.isUserInteractionEnabled = false
+    textField.backgroundColor = UIColor.black
+
+    originalParent = visibleView.layer.superlayer
+
+    if let viewSuperlayer = visibleView.layer.superlayer {
+      viewSuperlayer.addSublayer(textField.layer)
+
+      if let firstTextFieldSublayer = textField.layer.sublayers?.first {
+        visibleView.layer.removeFromSuperlayer()
+        visibleView.layer.position = CGPoint(x: visibleView.bounds.width / 2, y: visibleView.bounds.height / 2)
+        firstTextFieldSublayer.addSublayer(visibleView.layer)
+      }
+    }
+
+    protectionTextField = textField
```

Comment:
I'm not sure if we can modify the view hierarchy like this without negative consequences. I'll consult this with the rest of the team 🫡